include/ck/tensor_operation/gpu/device/device_grouped_gemm_fixed_nk.hpp

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+#include <vector>
```

Comment:
Those two headers are unused, but you need `array`
```suggestion
#include <array>
```